CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 Scalyr Agent 2 Changes By Release
 =================================
 
+## 2.1.16 "TBD" - TBD
+
+<!---
+Packaged by Tomaz Muraus <tomaz@scalyr.com> on Dec 2, 2020 14:00 -0800
+--->
+
+Misc:
+* The default value for the `k8s_cri_query_filesystem` kubernetes monitor configuration option has changed to `True`. This means that by default when in CRI mode, the monitor will only query the filesystem for the list of active containers, rather than first querying the Kubelet API. If you wish to query the Kubelet API for active containers configure `k8s_cri_query_filesystem` as false.
```

Comment:
Since this is for K8s, I would document how to revert by using the environment vars.  I believe the environment var for this would be `SCALYR_K8S_CRI_QUERY_FILESYSTEM` but double check that.  

I would also say, "If you wish to revert to the original default to prefer using the Kubelet API, set `SCALYR_K8S_CRI_QUERY_FILESYSTEM` the environment variable to "false" for the Scalyr Agent daemonset.